### PR TITLE
Sldss 25520 extend public api docs for issues

### PR DESCRIPTION
**Pull request by achebanets**

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -3809,6 +3811,51 @@ paths:
           curl -X GET -H "Authorization: Bearer $smartlingToken" -H
           "Content-Type: application/json"
           https://api.smartling.com/issues-api/v2/projects/$smartlingProjectId/issues/$issueUid/comments/$issueCommentUid
+    delete:
+      summary: Delete comment
+      description: >-
+        Delete comment text.
```
Comment (AlexArcher) [inline spec/openapi.yaml]: It deletes whole comment, not just text

File: spec/openapi.yaml [hunk 2]
```diff
@@ -3809,6 +3811,51 @@ paths:
...
+
+        **Note**: The author of an issue comment, account owner and project manager can perform this action.
```
Comment (AlexArcher) [inline spec/openapi.yaml]: an > the

File: spec/openapi.yaml (hunk 1 above)
Comment (achebanets) [reply spec/openapi.yaml]: Done

File: spec/openapi.yaml (hunk 2 above)
Comment (achebanets) [reply spec/openapi.yaml]: Done

**Pull request merged**